README.md

```diff
@@ -1,5 +1,8 @@
 # storyblok-api
 
+> [!WARNING]
+> This Bundle is experimental and subject to change in a future release.
```

Comment:
```suggestion
> This Bundle is in the initial phase and subject to change in a future release.
```

@silasjoisten @Edo-San what do you think about this suggestion? I'd like to avoid the word "experimental" as it can give the feeling that is not production-ready, but I'm very sure it's not the case

Reply:
Sure go for it